spec/declaration.dd

```diff
@@ -348,6 +348,7 @@ $(GNAME AliasDeclarationX):
 $(GNAME AliasDeclarationY):
     $(GLINK_LEX Identifier) $(GLINK2 template, TemplateParameters)$(OPT) $(D =) $(GLINK StorageClasses)$(OPT) $(GLINK Type)
     $(GLINK_LEX Identifier) $(GLINK2 template, TemplateParameters)$(OPT) $(D =) $(GLINK2 expression, FunctionLiteral)
+    $(GLINK_LEX Identifier) $(GLINK2 template, TemplateParameters)$(OPT) $(D =) $(GLINK StorageClasses)$(OPT) $(GLINK BasicType) $(GLINK Parameters) $(GLINK MemberFunctionAttributes)$(OPT)
```

Comment:
The `Parameters` link doesn't work as that part of the grammar is not defined in the current page. Try the following: `$(GLINK2 function, Parameters)`. Ditto for `MemberFunctionAttributes`.

Reply:
Yeah thanks, strange that the CI doesn't catch invalid links BTW.